lib/pure/cookies.nim

```diff
@@ -9,18 +9,25 @@
 
 ## This module implements helper procs for parsing Cookies.
 
-import strtabs, times
+import std/[strtabs, times, options]
+
+
+type
+  SameSite* {.pure.} = enum ## The SameSite cookie attribute.
+                            ## `Default` means that `setCookie`
+                            ## proc will not set `SameSite` attribute.
+    Default, None, Lax, Strict
```

Comment:
std/compilesettings uses lower-case enum values, this is inconsistent; we should specify in nep1 which one is the "best practice"

Reply:
It is already specified in nep1

![image](https://user-images.githubusercontent.com/43030857/109251396-5ed46400-7826-11eb-82e2-42b26e870039.png)